recipes/py_ecc/LICENSE

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
This file is in the tarball. You don't need to add it here again.

Reply:
right, thanks. Fixed.